titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/KubePodConfiguration.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.master.kubernetes.pod;
+
+import java.util.List;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+import com.netflix.titus.master.kubernetes.pod.resourcepool.CapacityGroupPodResourcePoolResolver;
+
+@Configuration(prefix = "titusMaster.kubernetes.pod")
```

Comment:
Should we keep the same prefix as DirectKubeConfiguraton so we don't have to verify any existing overrides we may have created?

Reply:
I would rather migrate them over to the new name instead of doing it later.